hs_model_program/templates/pages/modelprogramresource.html

```diff
@@ -13,59 +13,58 @@
                     {########################}
                     {#       CONTENT FILES     #}
                     {########################}
-
-                    {% if mpmetadata.get_engine_list.items|first|first or mpmetadata.get_software_list.items|first|first or mpmetadata.get_releasenotes_list.items|first|first or mpmetadata.get_documentation_list.items|first|first %}
+                    {% if mpmetadata.get_engine_list|first or mpmetadata.get_software_list|first or mpmetadata.get_releasenotes_list|first  or mpmetadata.get_documentation_list|first  %}
                     <legend>Content files</legend>
                     <table class="custom-table">
                         <tbody>
-                        {% if mpmetadata.get_engine_list.items|first|first %}
-                            {% for name, path in mpmetadata.get_engine_list.items %}
+                        {% if mpmetadata.get_engine_list|first %}
```

Comment:
Without this `if` condition, the "Model Engine" (or "Model Software" etc.) heading will appear without any file names there. For example, 
![image](https://cloud.githubusercontent.com/assets/6943441/25294252/55f3bb94-26ac-11e7-8b73-8d570bb82dd7.png)